src/database/schema/ChatSchema.ts

```diff
@@ -4,6 +4,7 @@ import {v4 as uuid} from 'uuid';
 import BaseDbSchema from './BaseDbSchema';
 import UserSchema from './UserSchema';
 import {ModifyAnonymousChatData} from '../../../types/repo/AnonymousMessageRepo/InitAnonymousChatData';
+import ChannelListSchema from './ChannelListSchema';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ChannelListSchema` is added, which seems to be used in the new method `getPendingMessages`. This change looks good as long as `ChannelListSchema` is properly exported from its module.
```diff
+ import ChannelListSchema from './ChannelListSchema';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->